packages/wonder-blocks-popover/src/components/__docs__/popover-content.argtypes.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import * as React from "react";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {Spring, Strut} from "@khanacademy/wonder-blocks-layout";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+const ActionsMappings: {[key: string]: React.Node} = {
+    singleButton: <Button kind="primary">Continue</Button>,
+    singleButtonEmphasized: (
+        <Button kind="primary" light={true}>
+            Continue
+        </Button>
+    ),
+    pagination: (
+        <>
+            <Button kind="tertiary">Previous</Button>
+            <Strut size={Spacing.medium_16} />
+            <Button kind="primary">Next</Button>
+        </>
+    ),
+    paginationEmphasized: (
+        <>
+            <Button kind="tertiary" light={true}>
+                Previous
+            </Button>
+            <Strut size={Spacing.medium_16} />
+            <Button kind="primary" light={true}>
+                Next
+            </Button>
+        </>
+    ),
+    steps: (
+        <>
+            <LabelLarge>Step 1 of 5</LabelLarge>
+            <Spring />
+            <Button kind="tertiary">Skip this step</Button>
+        </>
+    ),
+    stepsEmphasized: (
+        <>
+            <LabelLarge>Step 1 of 5</LabelLarge>
+            <Spring />
+            <Button kind="secondary" light={true}>
+                Skip this step
+            </Button>
+        </>
+    ),
+};
+
+export default {
+    content: {
+        description: "The content to render inside the popover.",
+        type: {required: true, summary: "string"},
+    },
+    title: {
+        description: "The popover title",
+        type: {required: true, summary: "string"},
+    },
+    actions: {
+        description: `User-defined actions.\n\nIt can be either a Node or a function using the children-as-function pattern to pass a close function for use anywhere within the actions. This provides a lot of flexibility in terms of what actions may trigger the Popover to close the popover dialog`,
```

Comment:
Do you think you could specify that the `Emphasized` options are intended to be used when the `emphasized` prop is true? I found it confusing initially because if you just do this without setting the `emphasized` prop, it just shows white buttons on a white background and it doesn't look emphasized. Maybe `detail` could be appropriate for this.

Reply:
If I understand the intent correctly, then it would be more clear if they were named with "Light" at the end instead of "Emphasized"?

Reply:
I originally named them `*Light` but noticed that there was no connection with the `Emphasized` state. I'll try to provide a better label for the dropdown.